benchmark/operations/create_container.py

```diff
@@ -0,0 +1,319 @@
+# Copyright 2016 ClusterHQ Inc.  See LICENSE file for details.
+"""
+Operation to create a container.
+"""
+
+from functools import partial
+from itertools import repeat
+import random
+from uuid import UUID, uuid4
+
+from pyrsistent import PClass, field
+from zope.interface import implementer
+
+from flocker.apiclient import MountedDataset
+from flocker.common import gather_deferreds, loop_until
+from flocker.control import DockerImage
+
+from .._interfaces import IProbe, IOperation
+
+
+class EmptyClusterError(Exception):
+    """
+    Exception indicating that the cluster contains no nodes.
+    """
```

Comment:
This hasn't come up in any previous operation? Seems like a fairly generic exception that
could affect a lot of operations. Is there somewhere more central to put it to be reused later?